faq/prooffacehelp.php

```diff
@@ -491,7 +491,6 @@ function echo_row($name, $tooltip, $button_image_base, $accelerator)
     echo_row('Show All Text', 'Show All Text', 'bt9', '');
     echo_row('Undo Revert', 'Undo Revert', 'bt7', '');
     echo_row('Revert to Original Document', 'Revert to Original Document', 'bt8', '');
-    echo_row('Refresh Image', 'Refresh Image', 'bt6', '');
     echo_row('Set Image Zoom Percent', 'Set Image Zoom Percent', 'bt10', ''); ?>
```

Comment:
I would defer updating `prooffacehelp`. I'm dubious how useful that is anyway and it's unclear how we would offer links to it within the Image Viewer Widget.

We do need to pull out the image zoom from userprefs, the initial DB schema, and a DB migration script as part of this. I can help with that if we want to do merge this into a temporary branch instead of `master`.

Reply:
> I would defer updating `prooffacehelp`. I'm dubious how useful that is anyway and it's unclear how we would offer links to it within the Image Viewer Widget.

I think that the image-related stuff that will no longer be available does need to be pulled out, because those buttons/links won't be in the interfaces anymore. Other than that, I agree -- I think at a minimum, we can wait.

Even if we do decide to update `prooffacehelp` to describe the Image Viewer Widget, though, I don't think we'd need to provide links to it within the widget. As far as I know, the only place on a proofing page that links to it is the `Help` link in the documentation section of the Toolbox. All that would be needed is updating the descriptions of the functionality. But I agree that I don't know how useful it would be. All of the buttons in the widget have tooltips, and are pretty self-explanatory.

Reply:
I was unclear, sorry about that. I agree we should remove references to the old controls but do not think we should add docs on new controls.

Reply:
> `Set Image Zoom Percent` needs to go, too. -- that icon and function no longer exist where they are.
> 
I've removed this and also the texts that go with the removed lines.

> I believe that the corresponding image manipulation function for the standard interface needs to come out, too (approximagely lines 426-438).
> 
Done this also.

> `<tb>`
> 
> Related, but not in this MR -- should the image zoom come out of the userprefs, or is that still used to set initial zoom settings for initial sessions in a new project? A really quick test (not exhaustive) seems to indicate that it goes by the existing local setting, not by what's in the userprefs, so should probably come out of the userprefs?

The setting is not used in the image widget but it is still used in md_phase2.php. I don't know what to do about it. Possible options:
1. enable metadata temporarily and consider whether to use the image widget in md_phase2.php.
2. remove all the metadata code.
3. something else.

Reply:
In a perfect world we would do option 1. However, I have no confidence that the metadata code actually works and I don't think it's worthwhile to spend time figuring out if it does just to do that option.

My suggestion is that we update `md_phase2.php` to set the initial zoom (`$iWidth`) to the default values for those fields and be done with it.

Reply:
My impression was that the `md_phase2`-related code was incomplete, and certainly hasn't been maintained over the years. -- I like @cpeel's suggestion.

Reply:
Done this.
Should we also remove the database entries?

Reply:
I would think it would be nice to clear up database, but that's my not very strong opinion.

Reply:
Yes, we need to update `db_schema.sql` and have an upgrade script that drops the columns from the DB.

Reply:
I've updated `db_schema.sql` and made an upgrade script by modifying existing models.